packages/web-components/src/components/masthead/__stories__/masthead.stories.ts

```diff
@@ -92,7 +91,7 @@ export const WithCustomNavigation = ({ parameters }) => {
     <style>
       ${styles}
     </style>
-    <dds-masthead-container-search-only
+    <dds-masthead-container
```

Comment:
@IgnacioBecerra if we remove `dds-masthead-container-search-only` and set it to just `dds-masthead-container`, the Custom Navigation menu items get overwritten from the data in the translation files. Let's switch back to `dds-masthead-composite`?